### PR TITLE
Fantapop improve examples

### PR DESCRIPTION
**Pull request by fantapop**
**Commit checklist**
- [x] Changelog
- [x] Doc gen (`make generate`)
- [ ] Integration test(s)
- [ ] Acceptance test(s)
- [x] Example(s)

### CODE REVIEW

File: examples/resources/cockroach_serverless_cluster/resource.tf [hunk 1]
```diff
@@ -1,12 +0,0 @@
-resource "cockroach_cluster" "cockroach" {
```
Comment (carloruiz) [inline examples/resources/cockroach_serverless_cluster/resource.tf:1]: Why is this removed?

**Review by carloruiz — APPROVED** [review]
nice

File: examples/resources/cockroach_serverless_cluster/resource.tf (hunk 1 above)
Comment (fantapop) [reply examples/resources/cockroach_serverless_cluster/resource.tf:1]: The cockroach_serverless_cluster and cockroach_dedicated_cluster examples were merged into one.  This is because the example directory name must match the resource in order for it to be included in the docs.  

See here: https://github.com/cockroachdb/terraform-provider-cockroach/pull/195/files#diff-9fbf857764cbe159f98d70fb75ba7f82ede6c04edd00ff216538db4e64ac1d6e

And the generated docs here:
https://github.com/cockroachdb/terraform-provider-cockroach/pull/195/files#diff-ea87985827a5e33692d9699416d9144971b191e9688a542125e8f31b5133adbd
